backend/app/views/spree/admin/payments/index.html.erb

```diff
@@ -3,7 +3,9 @@
 <% content_for :page_actions do %>
   <% if @order.outstanding_balance? %>
     <li id="new_payment_section">
-      <%= link_to t('spree.new_payment'), new_admin_order_payment_url(@order), class: 'btn btn-primary' %>
+      <% if can? :manage, Spree::Payment %>
```

Comment:
why `manage` and not `create`?

Reply:
@kennyadsl You are right here. Create should be enough. I'll fix it.